src/detection/cpu/cpu.h

```diff
@@ -15,6 +15,7 @@ typedef struct FFCPUResult
     FFstrbuf name;
     FFstrbuf vendor;
 
+    uint16_t cpuCount;
```

Comment:
Should this be renamed to something else like packageCount ?

Reply:
Not sure if `physical id` in Linux is the same as package in Windows and macOS. I guess cpuCount is ok.